examples/CompilerExtensions/Demo/Demo.csproj

```diff
@@ -1,9 +1,9 @@
-﻿<Project Sdk="Microsoft.Quantum.Sdk/0.15.2102129448">
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.22.189218-beta">
```

Comment:
shouldn't this be 0.23 too?

Reply:
Ah, that's interesting. It seems those weren't changed automatically by the build commit (06dbbdd1), but I've fixed them manually.